pandas/tests/generic/test_generic.py

```diff
@@ -654,6 +657,24 @@ def test_sample(sel):
         with pytest.raises(ValueError):
             df.sample(1, weights=s4)
 
+        # Check that random_state arguments are processed correctly
```

Comment:
can you make this a separate test & parameterize

Reply:
Done :+1: